frontend/src/components/workload/Overview.tsx

```diff
@@ -102,7 +108,7 @@ export default function Overview() {
               <WorkloadCircleChart
                 workloadData={workloadsData[name] || null}
                 // @todo: Use a plural from from the class itself when we have it
-                title={name + 's'}
+                title={ChartLink(name)}
```

Comment:
KubeObject.pluralName already has plural name available, we should use it here

Reply:
fixed!

Reply:
done!